src/rules/czech.toml

```diff
@@ -0,0 +1,16 @@
+allowed_symbols_regex="[A-Za-zěščřžýáíéóďťňĚŠČŘŽÝÁÍÉÓĎŤŇäöüÄÖÜ‚–\\. ]"
+min_trimmed_length = 6
+min_word_count = 1
+max_word_count = 14
+min_characters = 0
+may_end_with_colon = false
+quote_start_with_letter = true
+needs_punctuation_end = false
+needs_letter_start = true
+needs_uppercase_start = true
+even_symbols = ["\""]
```

Comment:
minor: given that the `allowed_symbols_regex` rule does not allow these quotes, this check will never be run, and could therefore be removed.

Reply:
OK, I am keeping them